affiliates.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Affiliates</title>
+</head>
+<body>
+    <section class="wrapper-pages">
+        <div class="container">
+            <div class="motto-wrap">
+                <div class="label-light">
+                    ONE FAMILY
+                </div>
+                <div class="affiliates-heading">
+                    Our Affiliates <br>
+                </div>
+            </div>
+            <div class="h-rule"></div>
+            <div class="affiliates-content-wrap">
+                <div class="affiliates-layout-grid affiliates-about-grid">
```

Comment:
this grid section has only children div instead of two. as a result map is stacked downwards